### ISSUE TITLE
Enhancement List

### ISSUE THREAD

**Issue by hey24sheep** [beemdevelopment/Aegis#583]
Features/Enhancement List

- File import documentation (file extension for different type of imports and how to export/import) Check #582 
- Json import
- Scan image how to? Which app is supported?
- Toast/UI feedback for manual backup (no visual feedback right now)
- What's the difference between Vault Export and Manual Trigger Backup?
- Chrome/Desktop App

**Comment by alexbakker** [comment]
> Json import

What does this mean?

> Scan image how to? Which app is supported?

As the store listing says, Aegis is compatible with Google Authenticator.

> What's the difference between Vault Export and Manual Trigger Backup?

There is no difference.

> Chrome/Desktop App

We don't have plans to port Aegis to other platforms.

Could you create separate issues for 1 and 4? Tracking multiple feature requests in one place will quickly become a bit messy.

**Comment by michaelschattgen** [comment]
>Scan image how to? Which app is supported?

As you've said in #582 you get the "No apps can perform this action" toast. This means Aegis can't find the correct document / image picker to choose your image from. Did you delete any of the DocumentUI or filepicker apps from your phone? Do you have a default gallery app set?

**Comment by hey24sheep** [comment]
@michaelschattgen No, I have default apps including gallery as well as Google Photos and Files app. That is why I thought this feature wants something else to work. I have gallery set as default for all pickers. It works in other apps without any issues. My phone is Honor Play with stock Android 9 on latest patch, if it helps in debugging.

**Comment by hey24sheep** [comment]
@alexbakker 

> > Json import
> 
> What does this mean?

I had a json file like this, using [this](https://github.com/wmdebrito/googleOTP-to-andOTP). So a documentation link with Json Import option would have helped. Like - 'JSON Format' with example file like mine

```
[
    {
        "algorithm": "SHA1",
        "digits": "6",
        "type": "TOTP",
        "secret": "...secret...",
        "label": "Google:account@gmail.com",
        "issuer": "Google",
        "thumbnail": "Google"
    }
]
```

> > Scan image how to? Which app is supported?
> 
> As the store listing says, Aegis is compatible with Google Authenticator.

As listed in my other closed issue #582  I am unable to scan from image.

> > What's the difference between Vault Export and Manual Trigger Backup?
> 
> There is no difference.

Could you add this to documentation?

> > Chrome/Desktop App
> 
> We don't have plans to port Aegis to other platforms.

Okay

**Comment by alexbakker** [comment]
Thanks for filing those new issues. I've marked them with ``proposal-accepted``. I'll close this one.

> I had a json file like this, using this. So a documentation link with Json Import option would have helped. Like - 'JSON Format' with example file like mine

That's andOTP's format.

> As listed in my other closed issue #582 I am unable to scan from image.

There's an existing issue about this: #538.
